packages/runtime/runtime-definitions/src/compatibilityBase.ts

```diff
@@ -84,151 +84,17 @@ export type ConfigMap<T extends Record<string, unknown>> = {
-		// `minVersionForCollab` introduction, which could be unexpected.
-		// Only enable as a default when `minVersionForCollab` is specified at
-		// 2.0.0+.
-		"2.0.0": true,
-	},
-	flushMode: {
-		// Note: 1.x clients are compatible with TurnBased flushing, but here we elect to remain on Immediate flush mode
-		// as a work-around for inability to send batches larger than 1Mb. Immediate flushing keeps batches smaller as
-		// fewer messages will be included per flush.
-		"1.0.0": FlushMode.Immediate,
-		"2.0.0-defaults": FlushMode.TurnBased,
-	},
-	gcOptions: {
-		"1.0.0": {},
-		// Although sweep is supported in 2.x, it is disabled by default until minVersionForCollab>=3.0.0 to be extra safe.
-		"3.0.0": { enableGCSweep: true },
-	},
-	createBlobPayloadPending: {
-		// This feature is new and disabled by default. In the future we will enable it by default, but we have not
-		// closed on the version where that will happen yet.  Probably a .10 release since blob functionality is not
-		// exposed on the `@public` API surface.
-		"1.0.0": undefined,
-	},
-} as const satisfies ConfigMap<RuntimeOptionsAffectingDocSchema>;
-
-const runtimeOptionsAffectingDocSchemaConfigValidationMap = {
-	enableGroupedBatching: configValueToMinVersionForCollab([
-		[false, "1.0.0"],
-		[true, "2.0.0-defaults"],
-	]),
-	compressionOptions: configValueToMinVersionForCollab([
-		[{ ...disabledCompressionConfig }, "1.0.0"],
-		[{ ...enabledCompressionConfig }, "2.0.0-defaults"],
-	]),
-	enableRuntimeIdCompressor: configValueToMinVersionForCollab([
-		[undefined, "1.0.0"],
-		["on", "2.0.0-defaults"],
-		["delayed", "2.0.0-defaults"],
-	]),
-	explicitSchemaControl: configValueToMinVersionForCollab([
-		[false, "1.0.0"],
-		[true, "2.0.0-defaults"],
-	]),
-	flushMode: configValueToMinVersionForCollab([
-		[FlushMode.Immediate, "1.0.0"],
-		[FlushMode.TurnBased, "2.0.0-defaults"],
-	]),
-	gcOptions: configValueToMinVersionForCollab([
-		[{ enableGCSweep: undefined }, "1.0.0"],
-		[{ enableGCSweep: true }, "2.0.0-defaults"],
-	]),
-	createBlobPayloadPending: configValueToMinVersionForCollab([
-		[undefined, "1.0.0"],
-		[true, "2.40.0"],
-	]),
-} as const satisfies ConfigValidationMap<RuntimeOptionsAffectingDocSchema>;
-
-/**
- * Returns the default RuntimeOptionsAffectingDocSchema configuration for a given minVersionForCollab.
- */
-export function getMinVersionForCollabDefaults(
-	minVersionForCollab: MinimumVersionForCollab,
-): RuntimeOptionsAffectingDocSchema {
-	return getConfigsForMinVersionForCollab(
-		minVersionForCollab,
-		runtimeOptionsAffectingDocSchemaConfigMap,
-		// This is a bad cast away from Partial that getConfigsForCompatMode provides.
-		// ConfigMap should be restructured to provide RuntimeOptionsAffectingDocSchema guarantee.
-	) as RuntimeOptionsAffectingDocSchema;
-}
-
 /**
  * Returns a default configuration given minVersionForCollab and configuration version map.
+ *
+ * @internal
```

Comment:
These functions are utilities not defintions. I believe runtime-utils is the right place for them.